apps/extension/entrypoints/twitter-observer.content/twitter-observer.tsx

```diff
@@ -3,9 +3,12 @@ import { setupTwitterObserver } from "@repo/tlinks/ext/twitter"
 import "@repo/tlinks/index.css"
 import { useEffect, useState } from "react"
 import "~/assets/style.css"
+import {TokenscriptCardMetadata} from "@repo/tlinks/dist/utils/index";
```

Comment:
please update the import path here, can treat `@repo/tlinks` as a normal npm package

Reply:
@futantan should be okay now I think?